frontend/src/utils/get.ts

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+
+export default function get<T>(route: string, setState: (data: T[]) => void) {
+  axios
+    .get(`http://localhost:8080${route}`)
```

Comment:
Maybe make the backend URL a constant in this file, so we don't have to update it for all of the HTTP methods